spec/request/admin/users_controller_spec.rb

```diff
@@ -111,6 +111,17 @@
       expect(users(:arthur).topics.count).to eq(0)
       expect(users(:arthur).comments.count).to eq(0)
     end
+
+    it "soft deletes from other user's playlist" do
+      # sudara's playlist
+      playlists(:owp)
+      # arthur's track on owp playlist
+      tracks(:track_on_soft_deleted_of_other_user)
+
+      put delete_admin_user_path(users(:arthur))
+
+      expect(tracks(:track_on_soft_deleted_of_other_user).reload.deleted_at).not_to be_nil
```

Comment:
[Change Matcher](https://relishapp.com/rspec/rspec-expectations/v/3-8/docs/built-in-matchers/change-matcher) might be your friend and prevent the need for reload call:

expect { put delete_admin_user_path(users(:arthur)) }.to change(track.deleted_at)